Automattic/wp-calypso
# Plugins plans: Update to show mobile view

#### Proposed Changes

Following #67891, needs mobile view for plugins plans.

#### Testing Instructions

* Go to `/plugins/plans/:site` using mobile view
* Plans should look like the `After` screenshot.

#### Screenshot
| Before | After | 
| --- | --- |
| <img width="285" alt="image" src="https://user-images.githubusercontent.com/402286/191162526-bbf1a0c7-2bd6-4978-a5f6-c5e1e6a35425.png"> | <img width="284" alt="image" src="https://user-images.githubusercontent.com/402286/191162468-11f87d36-70d6-4690-b482-8f935514d778.png"> |



#### Pre-merge Checklist



Fixes #67961
